### ISSUE TITLE
Credentials

### ISSUE THREAD

**Issue by efrecon** [efrecon/influx-backup#4]
Provide a way to specify credentials for databases which access is restricted to given users. See error example below:

````
Connection to influx:8086 established
Getting list of databases
Backup metadata into /backup/20180205-142937
!! Cannot list databases: ERR: unable to parse authentication credentials
````

**Comment by efrecon** [comment]
Note that a work-around is to specify these through the `-influx` command-line option, but this would reveal credentials to eavesdroppers.

**Comment by efrecon** [comment]
Fixed as of c46c6cdf14a669da2d39950b40435e6c6e7bb20d
